src/test/java/picard/util/MiscTest.java

```diff
@@ -0,0 +1,26 @@
+package picard.util;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+
+public class MiscTest {
+
+    @Test
+    public void canWriteToDevNull() throws IOException {
```

Comment:
What is this testing? It doesn't cover any code that we've written. It can't assert anything about the behavior of a system picard will be run on, only the behavior of the system where the test is run.

Reply:
This was at the bottom of the Fingerprinting test for some reason...so I moved it out....I think that at some point in the past there were questions about the ability to write to /dev/null and so a test was added (possibly by me) in the wrong place...

Reply:
Ah, I see the problem..I forgot to remove it from where I found it..so it looks like I just added it out of the blue...